Templates/SasToken-Node/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "webHookType": "",
```

Comment:
nit: remove this uneccessary webHookType (unless you put it in there intentionally for a reason). Our default http templates don't include this.